docs/setup-cluster/k8s/setup-multiple-resource-managers.rst

```diff
@@ -0,0 +1,167 @@
+.. _multiple-resource-managers:
+
+######################################
+ Configure Multiple Resource Managers
+######################################
+
+.. meta::
+   :description: Discover how to configure and manage multiple resource managers.
+
+.. include:: ../../_shared/attn-enterprise-edition.txt
+
+**********
+ Overview
+**********
+
+Multiple Resource Managers (Multi-RM) for Kubernetes allows you to set up a Determined master
+service in one Kubernetes cluster and schedule workloads in the same or other Kubernetes clusters.
+
+**Resource Pool Relationships**
+
+-  Resource pools have a many-to-one relationship with resource managers.
+-  No single resource pool will span multiple resource managers.
+
+Multiple resource managers are defined in the :ref:`master configuration <master-config-reference>`
+using the ``additional_resource_managers`` and ``resource_pools`` options. Any requests to resource
+pools not defined in the master configuration are routed to the default resource manager. Such
+requests are not routed to additional resource managers, if defined.
+
+*********************************************
+ How to Configure Multiple Resource Managers
+*********************************************
+
+To begin, you'll need a user with the Cluster Admin role (requires Determined Enterprise Edition).
+The Cluster Admin role is required for configuring multiple resource managers.
+
+.. attention::
+
+   **Naming and Rules**
+
+   -  Each resource manager under ``additional_resource_managers`` must have a unique name; failure
+      to do so will cause the cluster to crash.
+
+   -  Ensure each additional resource manager has at least one resource pool defined.
+
+   -  Resource pool names must be unique across the cluster to prevent crashes.
+
+.. note::
+
+   The default resource manager is available by default and does not require a specific name.
+
+#. Locate the ``resource_manager`` section in the :ref:`master configuration <master-config-reference>`
+   yaml file. This represents the default resource manager.
+#. Add ``additional_resource_managers`` under the ``resource_manager`` to configure extra resource
+   managers.
+#. Under ``additional_resource_managers``, define ``resource_pools`` for each additional resource
+   manager.
+
+
+Example: Master Configuration (devcluster)
+==========================================
+
+Follow this example to create as many resource managers (clusters) as needed.
```

Comment:
Note: can we add a reference to the files changed here https://github.com/determined-ai/determined/pull/9151? it's another good place to look